manifests/rsync/params.pp

```diff
@@ -10,11 +10,17 @@
   case $::osfamily {
     openbsd: {
       include sys::openbsd::pkg
+      $provider = undef
       $source = $sys::openbsd::pkg::source
     }
     solaris: {
       include sys::solaris
       $provider = 'pkg'
+      $source = undef
+    }
+    default: {
+      $provider = undef
+      $source = undef
```

Comment:
Interesting, I wonder if this is a behavior that changed in puppet 4 or 5. Because this works fine on puppet 3.

Reply:
I suspect it's because I have strict variables checking enabled by default. I thought it had been made the default in one of the major Puppet releases, but apparently not (yet).